common/common.go

```diff
@@ -36,6 +36,7 @@ type BatchRun struct {
 	ProjectName      string `json:"project_name"`
 	BatchRunNumber   int    `json:"batch_run_number"`
 	TestSettingName  string `json:"test_setting_name"`
+	BranchName  string `json:"branch_name"`
```

Comment:
[nit] Perhaps this file's original author wants the maintainer to align the field names:

```suggestion
	BranchName       string `json:"branch_name"`
```